src/main/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorCreateApplication.java

```diff
@@ -109,8 +109,8 @@ private PageBuilder addEventConfig(
             .endButtonLabel("Save Application")
             .aboutToSubmitCallback(this::aboutToSubmit)
             .explicitGrants()
-            .grant(CREATE_READ_UPDATE, CASEWORKER_DIVORCE_SOLICITOR)
+            .grant(CREATE_READ_UPDATE, CASEWORKER_DIVORCE_SOLICITOR, CASEWORKER_DIVORCE_COURTADMIN, CASEWORKER_DIVORCE_COURTADMIN_BETA)
```

Comment:
do we need court admin role to have CRD for solicitor create ?

Reply:
This is a work around, until we have a fix for the generator authorisation,  if I don't add caseworker access here then the caseworker is not able to upload documents.  Ideally we need a finer grain control over the authorisation on single fields.